nixos/atticd.nix

```diff
@@ -51,30 +59,30 @@ let
       ${atticadmShim} "$@"
   '';
 
-  hasLocalPostgresDB = let
-    url = cfg.settings.database.url or "";
-    localStrings = [ "localhost" "127.0.0.1" "/run/postgresql" ];
-    hasLocalStrings = lib.any (lib.flip lib.hasInfix url) localStrings;
-  in config.services.postgresql.enable && lib.hasPrefix "postgresql://" url && hasLocalStrings;
+  hasLocalPostgresDB =
+    let
+      url = cfg.settings.database.url or "";
+      localStrings = [
+        "localhost"
+        "127.0.0.1"
+        "/run/postgresql"
+      ];
+      hasLocalStrings = lib.any (lib.flip lib.hasInfix url) localStrings;
+    in
+    config.services.postgresql.enable && lib.hasPrefix "postgresql://" url && hasLocalStrings;
 in
 {
+  imports = [
+    (lib.mkRemovedOptionModule [ "services" "atticd" "credentialsFile" ] "The option was renamed to `services.atticd.environmentFile` to follow the module in nixpkgs.")
```

Comment:
I wonder if this should be a `mkRenamedOptionModule` instead. I don't think, other than the name difference, any semantics actually changed, so it might be nicer to consumers to not make it a hard error and instead just a warning?

(I think `mkRenamed....` adds an eval warning... if I'm wrong, disregard)